targets/lenovo-x1/everything.nix

```diff
@@ -271,19 +187,21 @@
           lanzaboote.nixosModules.lanzaboote
           microvm.nixosModules.host
           disko.nixosModules.disko
-          (import ../modules/partitioning/lenovo-x1-disko-basic.nix {device = "/dev/nvme0n1";}) #TODO define device in hw def file
-          ../modules/partitioning/disko-basic-postboot.nix
-          ../modules/host
-          ../modules/virtualization/microvm/microvm-host.nix
-          ../modules/virtualization/microvm/netvm.nix
-          ../modules/virtualization/microvm/guivm.nix
-          ../modules/virtualization/microvm/appvm.nix
+          (import ../../modules/partitioning/lenovo-x1-disko-basic.nix {device = "/dev/nvme0n1";}) #TODO define device in hw def file
+          ../../modules/partitioning/disko-basic-postboot.nix
+          ../../modules/host
+          ../../modules/virtualization/microvm/microvm-host.nix
+          ../../modules/virtualization/microvm/netvm.nix
+          ../../modules/virtualization/microvm/guivm.nix
+          ../../modules/virtualization/microvm/appvm.nix
```

Comment:
I wonder if we could refactor out these kinds of imports that import from one branch to another. In my own NixOS configs I import my modules at flake root level and then pass them to other branches as arguments. The advantages of this would be that we don't get so wedded into our folder structure and arguably it would enhance readability as well. We could have `ghafModules` that we pass along.

Reply:
we can and we will. this patch set is called bit by bit for a reason :)
  
it will eventually become this
https://github.com/tiiuae/ghaf/pull/390/files#diff-24cbae53da3e4c7b899685295737b955daa048345f53ef85ab51b07270d7b31e